platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/XMLUtils.java

```diff
@@ -124,28 +135,79 @@ public static Result transform(Node nodeXml, TransformerProperties transformerPr
      */
     public static String getRootNamespace(String xml) {
 
-        String rootNamespace = null;
-
         if (xml == null) {
             return null;
         }
 
-        initializeXMLInputFactory();
+        return processElements(xml, (result, xmlStreamReader) -> {
+            result.set(xmlStreamReader.getNamespaceURI());
+            return false;
+        });
+    }
 
```

Comment:
The method `transformation` is not changed. It just got moved around by the code organizer.